### PR TITLE
Pass logLevel to the Cargohold YAML config

### PR DESCRIPTION
**Pull request by neongreen**

### CODE REVIEW

File: charts/cargohold/values.yaml [hunk 1]
```diff
@@ -37,3 +37,6 @@ resources:
 #       domain: assets.example.com
 #       keyPairId: ABC....
 #       privateKeyPath: /etc/wire/cargohold/secrets/cf-pk.pem
+
+config:
```
Comment (ChrisPenner) [inline charts/cargohold/values.yaml]: Maybe move this above the 'provide me please' block so it doesn't get lost in the shuffle.

File: charts/cargohold/templates/configmap.yaml [hunk 2]
```diff
@@ -4,6 +4,9 @@ metadata:
   name: "cargohold"
 data:
   cargohold.yaml: |
+    logNetStrings: True # log using netstrings encoding: http://cr.yp.to/proto/netstrings.txt
```
Comment (ChrisPenner) [inline charts/cargohold/templates/configmap.yaml]: Is this potentially something we'd ever want to change? If yes; we should add it to values.yaml; if no, then why is it configurable? Haha

Comment (neongreen) [reply charts/cargohold/templates/configmap.yaml]: > if no, then why is it configurable

We don't want netstrings logging in the demo, but we want it in production, and k8s is for production only. (Actually, is there a good reason to not run the demo with minikube or k3s locally?)

File: charts/cargohold/values.yaml (hunk 1 above)
Comment (neongreen) [reply charts/cargohold/values.yaml]: Done.

File: charts/cargohold/templates/configmap.yaml (hunk 2 above)
Comment (ChrisPenner) [reply charts/cargohold/templates/configmap.yaml]: Sounds good; I was just curious :) If you have enough memory on your machine I don't see why not; but that depends on how scaled down the demo is. Local k8s can be a pretty big resource drain.

**Pull request merged**